src/Configurations/Fields/Sections/General/useProvidersOptions.js

```diff
@@ -1,18 +1,7 @@
-import { useQuery } from 'react-query';
-
-import { useOkapiKy } from '@folio/stripes/core';
-
-const API = 'remote-storage/providers';
-const LIMIT = 1000;
+import { Providers } from '../../../../API';
 
 export const useProvidersOptions = (params) => {
```

Comment:
I'd suggest moving this hook to `/src/data/Providers` (kind of data layer) - and use it then as `Providers.useOptions()`